motionplan/tpspace/ptgDiffDrive.go

```diff
@@ -0,0 +1,74 @@
+package tpspace
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/golang/geo/r3"
+
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/spatialmath"
+)
+
+// ptgDiffDrive defines a PTG family composed of circular trajectories with an alpha-dependent radius.
+type ptgDiffDrive struct {
+	maxMMPS float64 // millimeters per second velocity to target
+	maxRPS  float64 // radians per second of rotation when driving at maxMMPS and turning at max turning radius
+}
+
+// NewDiffDrivePTG creates a new PTG of type ptgDiffDrive.
+func NewDiffDrivePTG(maxMMPS, maxRPS float64) PTG {
+	return &ptgDiffDrive{
+		maxMMPS: maxMMPS,
+		maxRPS:  maxRPS,
+	}
+}
+
+// For this particular driver, turns alpha into a linear + angular velocity. Linear is just max * fwd/back.
+// Note that this will NOT work as-is for 0-radius turning. Robots capable of turning in place will need to be special-cased
+// because they will have zero linear velocity through their turns, not max.
```

Comment:
I'm confused by this comment, since this PTG does support 0-radius turning.

Reply:
I wrote this file by starting with a duplicate of the circular PTG file and editing, I missed updating this comment.